[DECLUTTER USING] re-write cold wasabi (#591)
zkSNACKs/WasabiDoc: docs/using-wasabi/WalletGeneration.md

If you had a billion computers, each one a billion times stronger than the computers available today, you would still not be able to crack this password.
Keep a [secure backup](/using-wasabi/ColdWasabi.md#a-list-of-the-more-common-mediums-of-cold-storage-with-some-of-their-weaknesses) of the password, but store it in a separate location from your main 12 recovery words.

## Backup

A list of the more common mediums for backups with some of their weaknesses:

### Written on a piece of paper
- Anyone who can see it, can steal it
- Handwriting can be hard to read or completely illegible
- Human error in transcription can cause errors on end product
- Paper can rot, be torn, burn, or be smoke damaged

### Printed on a piece of paper
- Anyone who can see it, can steal it

> - Type of printer - non-laser printers can run if paper gets wet

Reviewer: ```suggestion
- Type of printer - non-laser printers ink can run if the paper gets wet
```

- Have to trust printer - some have internet connections, wifi, and memory
- Paper can rot, be torn, burn, or be smoke damaged

### On laminated paper
- Anyone who can see it, can steal it

> - Lamination is prone or degradation over time and puncture or cuts that could allow moisture to get trapped in the paper and cause deterioration or rotting in some circumstances | store in cool dry place

Reviewer: ```suggestion
- Lamination is prone to degradation over time, as well as punctures or cuts that could allow moisture to get trapped in the paper and cause deterioration or rotting in some circumstances | store in cool dry place
```

So magnesium, tin, and lead are all out as engraving materials.
- Silver, gold, copper, brass, bronze, nickel, cobalt, would survive the housefire fire unmelted.
Some Aluminium alloys can survive but you have to have the right ones.
At around 1500° Steel and Nickel should be okay.
Titanium is above the housefire range and so is tungsten, however tungsten rings are known to shatter due to the brittle nature of the very hard metal.

### Stored digitally on a computer
- Computers can crash, making data recovery expensive
- Data can still technically be recovered after a system is abandoned by the user. In some cases data can be recovered after multiple overwriting attempts and physical destruction (as long as the attacker can get all or most the pieces) so if you copy files to a new computer and ditch the old one, be careful.
- Can burn or be smoke damaged
- A traditional hard disc drive can have data corrupted by powerful magnetic fields and can physically shatter
- A non-negligible amount of HDDs suffer from factory defects that will cause them to fail unexpectedly during their - lifetime
- Accidents can happen that could result in loss of data

> - Solid state drives (SSDs) will lose data if unpowered, they may last years before this becomes a problem but it is unwise to store long-term data in unpowered SSDs

Reviewer: Is this true? I have several SSD's that I use for various backups and they are fine without power
```suggestion
- Solid state drives (SSDs) will lose data if unpowered, they may last years before this becomes a problem but it is unwise to store long-term data in unpowered SSDs
```
Is this true? I have several SSD's that I use for various backups and they are fine without power.

Author: I'm not sure... @PulpCattel wrote this, I believe.

Reviewer 2: Guys IMHO we shouldn't add things that we are not sure about.

> - If connected to internet it is another attack vector and the safety is only as good as the encryption used; I don't know what I would recommend but it wouldn't be BitLocker. Someone could be trying to break into the computer constantly. Even with good encryption if the machine or location is compromised the key could be stolen as soon as it is decrypted.

Reviewer: ```suggestion
- Connecting to the internet is another attack vector and the safety is only as good as the encryption used; I don't know what I would recommend but it wouldn't be BitLocker. Someone could be trying to break into the computer constantly. Even with good encryption if the machine or location is compromised the key could be stolen as soon as it is decrypted.
```

- There are a lot of ongoing threats with computers, from 0-day exploits to firmware exploits and malicious USB cords

> - External hdds are good for storage for a few years at least if stored properly

Reviewer: ```suggestion
- External HDDs are good for storage for a few years at least if stored properly
```

- If not connected to internet, safety is only as good as the physical protection encryption used; could someone break into the location and copy the data without anyone noticing?

### Stored digitally on CD, floppy disk, laserdisc, or mini-disc
- Plastics break down over time and with exposure to heat, humidity, regular light, all sorts of chemicals, even the oxygen in the air.
This can lead to the loss of your data when stored on a medium made of plastic or written/printed on plastic.
- Can burn or be smoke damaged
- Can be physically damaged, making data recovery expensive or even impossible
- Magnetic media (tapes, floppy disc) can be damaged by magnets
- Data can become difficult to recover if the software and/or hardware to decode is old, don't use proprietary formats

### Stored digitally on a flash drive
- Can break and have to be physically repaired before use
- Rapidly changing magnetic fields (See MRIs) can damage the data stored on flash drives

> - Can burn or be smoke damaged

Reviewer: ```suggestion
- Can burn, or be smoke damaged
```

- Can become corroded from salt water or some atmospheric conditions
- If they break apart, some lighting conditions can cause data corruption (you can also put them back together and often still get the data)

> - Different devices are all different, even similar devices from the same production batch can be different.

Reviewer: ```suggestion
- Different devices are inconsistent.
Even similar devices from the same production batch can be different.
```

> There are large quality differences in drives but I am assuming you aren't using these for anything but storage.

Reviewer: ```suggestion
There are large quality differences in drives, but I am assuming you aren't using these for anything but storage.
```

> - There are some fake flash drives that look like they saved the data but you can't get it back later

Reviewer: ```suggestion
- There are some fake flash drives that look like they saved the data but you can't get it back later.
```

If you had a billion computers, each one a billion times stronger than the computers available today, you would still not be able to crack this password.
Keep a [secure backup](/using-wasabi/ColdWasabi.md#a-list-of-the-more-common-mediums-of-cold-storage-with-some-of-their-weaknesses) of the password, but store it in a separate location from your main 12 recovery words.

> ## Backup

Reviewer 2: The title should be `Backup Weaknesses`
All the information in this section should be in a separate file and not in the `Using Wasabi` sub-menu but in the `Advanced Using Wasabi`.

Also after enumerating all the weaknesses it would be good to have a better backup method than what was mentioned.

Author: Yes, ACK.

I will move it in a follow up PR.

A list of the more common mediums for backups with some of their weaknesses:

### Written on a piece of paper
- Anyone who can see it, can steal it
- Handwriting can be hard to read or completely illegible
- Human error in transcription can cause errors on the end product
- Paper can rot, be torn, burn, or be smoke damaged

### Printed on a piece of paper
- Anyone who can see it, can steal it
- Type of printer - non-laser printers ink can run if the paper gets wet
- Have to trust printer - some have internet connections, wifi, and memory
- Paper can rot, be torn, burn, or be smoke damaged

### On laminated paper
- Anyone who can see it, can steal it
- Lamination is prone to degradation over time, as well as punctures or cuts that could allow moisture to get trapped in the paper and cause deterioration or rotting in some circumstances | store in cool dry place
- Can burn or be smoke damaged
- 'Fireproof' & 'Fire-resistant' boxes can help protect paper in a small house fire but be warned that they can sometimes fall apart in the fire and get wet if the fire is put out with water.
Remember people can just carry out a small safe

> ### Engraved / etched/ ablated/ stamped on a piece of metal

Reviewer 2: ```suggestion
### Engraved / etched / ablated / stamped on a piece of metal
```

- Anyone who can see it, can steal it

> - Some metals can deteriorate or corrode, choose a good metal; also store your metal away from direct contact other metals. - Some metals that are corrosion resistant have low melting points, are extremely expensive, or hard to machine.

Reviewer 2: ```suggestion
- Some metals can deteriorate or corrode, choose a good metal; also store your metal away from direct contact other metals.
- Some metals that are corrosion resistant have low melting points, are extremely expensive, or hard to machine.
```

> - Metals can still deform or melt from heat, destroying any engraved SK. "Most house fires do not burn hotter than 1,200 degrees Fahrenheit.

Reviewer 2: ```suggestion
- Metals can still deform or melt from heat, destroying any engraved SK.
"Most house fires do not burn hotter than 1,200 degrees Fahrenheit.
```

This temperature is typically associated with the hottest portion of a home, which is in the roof area.
Homes that burn for longer than 30 minutes or consist of multiple levels sometimes burn at higher temperatures."
You want to pick a metal that won't be destroyed by a fire.
So magnesium, tin, and lead are all out as engraving materials.

> - Silver, gold, copper, brass, bronze, nickel, cobalt, would survive the housefire fire unmelted.

Reviewer 2: ```suggestion
- Silver, gold, copper, brass, bronze, nickel, cobalt, would survive the housefire unmelted.
```

Some Aluminium alloys can survive but you have to have the right ones.
At around 1500° Steel and Nickel should be okay.
Titanium is above the housefire range and so is tungsten, however tungsten rings are known to shatter due to the brittle nature of the very hard metal.

### Stored digitally on a computer
- Computers can crash, making data recovery expensive

> - Data can still technically be recovered after a system is abandoned by the user. In some cases data can be recovered after multiple overwriting attempts and physical destruction (as long as the attacker can get all or most the pieces) so if you copy files to a new computer and ditch the old one, be careful.

Reviewer 2: ```suggestion
- Data can still technically be recovered after a system is abandoned by the user.
In some cases data can be recovered after multiple overwriting attempts and physical destruction (as long as the attacker can get all or most the of pieces) so if you copy files to a new computer and ditch the old one, be careful.
```

- Can burn or be smoke damaged
- A traditional hard disc drive can have data corrupted by powerful magnetic fields and can physically shatter

> - A non-negligible amount of HDDs suffer from factory defects that will cause them to fail unexpectedly during their - lifetime

Reviewer 2: ```suggestion
- A non-negligible amount of HDDs suffer from factory defects that will cause them to fail unexpectedly during their lifetime
```

- Accidents can happen that could result in loss of data
- Solid state drives (SSDs) will lose data if unpowered, they may last years before this becomes a problem but it is unwise to store long-term data in unpowered SSDs

> - Connecting to the internet is another attack vector and the safety is only as good as the encryption used; I don't know what I would recommend but it wouldn't be BitLocker. Someone could be trying to break into the computer constantly. Even with good encryption if the machine or location is compromised the key could be stolen as soon as it is decrypted.

Reviewer 2: ```suggestion
- Connecting to the internet is another attack vector and the safety is only as good as the encryption used; I don't know what I would recommend but it wouldn't be BitLocker.
Someone could be trying to break into the computer constantly.
Even with good encryption if the machine or location is compromised the key could be stolen as soon as it is decrypted.
```
